src/dynarec/dynarec_arm_66.c

```diff
@@ -787,7 +787,7 @@ uintptr_t dynarec66(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int ninst,
                 case 5:
                     INST_NAME("SHR Ed, Ib");
                     if(geted_ib(dyn, addr, ninst, nextop)&31) {
-                        SETFLAGS(X_OF|X_CF, SF_SUBSET_PENDING);
+                        SETFLAGS(X_SF|X_CF|X_ZF|X_PF, SF_SUBSET_PENDING);
```

Comment:
OF is still set. I guess X_ALL should be used. Also, I noticed SAR next is also wrong.

Reply:
> OF is still set. I guess X_ALL should be used. Also, I noticed SAR next is also wrong.

I consulted Intel's instruction manual and found that SHR count does not have the OF flag, instead SHR 1 does.[Intel+439](https://cdrdv2-public.intel.com/812392/325462-sdm-vol-1-2abcd-3abcd-4.pdf)

Reply:
Yes, I agree, SHR 1 set OF, for other SHR, OF is undefined, so it's ok to mark it as "set", else, the dynarec will try to get OF from previous opcodes, wich is not needed, as OF is undefined.

Reply:
> Yes, I agree, SHR 1 set OF, for other SHR, OF is undefined, so it's ok to mark it as "set", else, the dynarec will try to get OF from previous opcodes, wich is not needed, as OF is undefined.

Okay, I see, Thank you.